deploy/20_deployMarketplace.ts

```diff
@@ -6,7 +6,10 @@ import { deployments } from "hardhat";
 const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
```

Comment:
I am strongly against changing migrations that have been followed to production.
The outcome of doing so is an environment during tests that differs from production.
So all your tests might pass - but won't be testing what's in production.

In this case, you're deploying a `Marketplace` contract that has an extra facet in `test` environment...

So unless you make absolutelly 100% sure that you have a migration function to bring production to the same state, it's tricky.

How about having a migration which reflects the steps that will need to be taken in production, for example calling an upgrade and a separate migration function such as `migrateV1toV2`?